docs/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.text-classifier.md

```diff
@@ -0,0 +1,47 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+title: Text Classifier
+description: Documentation for the Text Classifier node in n8n, a workflow automation platform. Includes details of operations and configuration, and links to examples and credentials information.
+---
```

Comment:
```suggestion
contentType: integration
---
```

Reply:
Template was missing this metadata. It's there now but just wanted to be sure we didn't miss it here. :)